katello.te

```diff
@@ -35,6 +35,7 @@ require{
     type websockify_t;
     type candlepin_activemq_port_t;
     type tomcat_t;
+    type systemd_unit_file_t;
```

Comment:
Can you remove this commit, let's get it green, merge and I will rebase on top of that again.

Reply:
Or wait these are separate commits, we should be okay we can cherry pick it.